serde_derive/src/lib.rs

```diff
@@ -9,9 +9,455 @@
 //! # fn main() {}
 //! ```
 //!
-//! Please refer to [https://serde.rs/derive.html] for how to set this up.
+//! Please refer to [https://serde.rs/derive.html] for how to set this up and
+//! troubleshoot.
 //!
 //! [https://serde.rs/derive.html]: https://serde.rs/derive.html
+//!
+//! # Examples
+//!
+//! An example below while showing the differences between attributes.
```

Comment:
This is not a grammatical sentence.

Copying the intro from https://serde.rs/attributes.html would be sufficient I think.